src/python/dxpy/bindings/dxfile_functions.py

```diff
@@ -466,3 +466,50 @@ def upload_string(to_upload, media_type=None, keep_open=False, wait_on_close=Fal
         handler.close(block=wait_on_close, **remaining_kwargs)
 
     return handler
+
+def download_folder(project, destdir, folder="/", chunksize=dxfile.DEFAULT_BUFFER_SIZE):
```

Comment:
Please allow the caller to supply additional kwargs, and pass them through to download_dxfile (download_dxfile itself does this). These can be used to customize the low-level request parameters.